packages/ui5-middleware-onelogin/sample/example_embeddedflp/ui5.yaml

```diff
@@ -0,0 +1,37 @@
+specVersion: "2.2"
+metadata:
+  name: ui5-app
+type: application
+resources:
+  configuration:
+    paths:
+      webapp: ../../../ui5-app/webapp
+
+framework:
+  name: OpenUI5
+  version: "1.95.0"
```

Comment:
OpenUi5 version should be aligned with the ecosystem sample or at least with the `@openui5/ts-types` or `@openui5/ts-types-esm`

Reply:
changed to 1.97, however the ui5-app is using 1.95 as well. So maybe that should be changed as well?

Reply:
Yes, that would be great. 👍